jobs/cron/templates/bin/setup_crontab.erb

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -exu
+
+CONFIG_DIR=/var/vcap/jobs/cron/config
+
+mkdir -p ${CONFIG_DIR}
+rm -f ${CONFIG_DIR}/entry-*
+
+read -r -d '' CRONTAB <<'EOF'
+<% p("cron.variables", {}).each do |k, v| %>
+<%= k %>=<%= v %>
+<% end %>
+EOF
+
+<% p("cron.entries", []).each do |entry| %>
+<% if entry.key?("script") %>
+ENTRY=$(mktemp ${CONFIG_DIR}/entry-XXXXXX)
```

Comment:
My initial thought would be to do something like:

    script:
       name: foo.sh
       contents: | 
         #!/bin/bash
          ..

Although I would worry about people including absolute paths in the name and being confused when `name: /tmp/blah.sh` ends up  in `/var/vcap/jobs/cron/config/tmp/blah.sh`

WDYT?